config/management/README.md

```diff
@@ -19,7 +19,7 @@ The process for starting organization of the planned and current functionality i
   * Association sets up a secure-backend for data uploads, `association drive`.
     The association then distributes credentials for each node owner and
     validator operator.
-  * The association generates its `association key` and shares the public key
+  * The association generates its `libra root key` and shares the public key
```

Comment:
@davidiw, do we still refer to the association as "the association", or "libra root"? The name change makes sense for the key, but I'm not sure how much sense it makes to rename the association itself?

Reply:
I probably wouldn't rename the association

Reply:
libra rooooot